packages/kbn-pm/package.json

```diff
@@ -61,7 +61,7 @@
     "strip-ansi": "^4.0.0",
     "strong-log-transformer": "^2.1.0",
     "tempy": "^0.3.0",
-    "typescript": "3.7.2",
+    "typescript": "3.8.2",
```

Comment:
(Thinking) npm really need a maven-like pinable version management for multi-projects...